code/obj/item/plants.dm

```diff
@@ -421,11 +421,15 @@ ABSTRACT_TYPE(/obj/item/plant/herb)
 	Crossed(atom/movable/AM as mob|obj)
 		var/mob/M = AM
 		if(iswerewolf(M))
-			M.changeStatus("weakened", 3 SECONDS)
-			M.force_laydown_standup()
-			M.TakeDamage("All", 0, 5, 0, DAMAGE_BURN)
-			M.visible_message("<span class='alert'>The [M] steps too close to [src] and falls down!</span>")
-			return
+			var/stun_duration
+			if (!GET_COOLDOWN(M, "aconite_stun"))
```

Comment:
```suggestion
			if (ON_COOLDOWN(M, "aconite_stun", stun_duration))
```
`ON_COOLDOWN` is nice because it both sets and checks a cooldown- if the thing is already on cooldown it returns `FALSE` and doesn't do anything else, otherwise it sets the cooldown and returns `TRUE`